docs/source/users-guide.rst

```diff
@@ -2361,6 +2360,31 @@ And you can override relevant fields via environment variables as follows:
   FABRIC_CA_SERVER_BCCSP_PKCS11_PIN=98765432
   FABRIC_CA_SERVER_BCCSP_PKCS11_LABEL=ForFabric
 
+
+The prebuilt Hyperledger Fabric Docker images are not enabled to use PKCS11. If
+you are deploying the Fabric CA using docker, you need to build your own image
+and enable PKCS11 using the following command:
+
+.. code:: bash
+
+  make docker GO_TAGS=pkcs11
+
+You also need to ensure that the PKCS11 library is available to be used by the
+CA by installing it or mounting it inside the container. If you are deploying
+your Fabric CA using docker compose, you can update your compose files to mount
+the softhsm library and configuration file inside the container using volumes.
+As an example, you would add the following environment and volumes variables to
+your docker compose file:
```

Comment:
I think you need to make it more clear that the whole section on SoftHSM is an example:

```
You also need to ensure that the PKCS11 library is available to be used by the
CA by installing it or mounting it inside the container. 

If you are deploying
your Fabric CA using Docker Compose, you can update your Compose files to mount
the library and any provider specific configuration file(s) inside the container using volumes.

For example, you would add the following environment variables and volumes to
your Docker Compose in order to use SoftHSM:
```